build.gradle.kts

```diff
@@ -46,7 +46,6 @@ allprojects {
     version = project.versionName
 
     extensions.configure<KtlintExtension>("ktlint") {
-        version by "0.40.0"
```

Comment:
Please leave the `ktlint` version here - we'll probably want to bump the `ktlint` version independently of the plugin in the future.